### PR TITLE
test: Add integration test for upload_without_processing route

### PR DESCRIPTION
**Pull request by irinaschubert**
### Task Description/Number

So far, there hasn't been an integration test for the LUA route upload_without_processing. Now, there is one.

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] fix: represents bug fixes
- [ ] refactor: represents production code refactoring
- [ ] feat: represents a new feature
- [ ] docs: documentation changes (no production code change)
- [ ] chore: maintenance tasks (no production code change)
- [x] test: all about tests: adding, refactoring tests (no production code change)
- [ ] other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] Maybe (not 100% sure => check with FE)

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [x] No

### CODE REVIEW

File: webapi/src/it/scala/org/knora/webapi/it/v2/KnoraSipiIntegrationV2ITSpec.scala [hunk 1]
```diff
@@ -429,6 +432,67 @@ class KnoraSipiIntegrationV2ITSpec
...
+      val savedValues: JsonLDArray = getValuesFromResource(
+        resource = resource,
+        propertyIriInResult = OntologyConstants.KnoraApiV2Complex.HasStillImageFileValue.toSmartIri
+      )
+
+      val savedValue: JsonLDValue = if (savedValues.value.size == 1) {
+        savedValues.value.head
+      } else {
+        throw AssertionException(s"Expected one file value, got ${savedValues.value.size}")
+      }
```
Comment (mpro7) [inline webapi/src/it/scala/org/knora/webapi/it/v2/KnoraSipiIntegrationV2ITSpec.scala:483]: I'm not sure if this makes sense at all.

File: webapi/src/main/scala/org/knora/webapi/messages/store/sipimessages/SipiMessages.scala [hunk 2]
```diff
@@ -127,6 +127,19 @@ case class SipiUploadResponseEntry(
...
+ * Represents the information that Sipi returns about each file that has been uploaded to the upload_without_processing route.
+ *
+ * @param filename            the filename that was submitted to Sipi.
+ * @param checksum            the checksum of the file
+ * @param checksumAlgorithm   the algorithm that was used to create the checksum of the file
+ */
+case class SipiUploadWithoutProcessingResponseEntry(
+  filename: String,
+  checksum: String,
+  checksumAlgorithm: String
```
Comment (mpro7) [inline webapi/src/main/scala/org/knora/webapi/messages/store/sipimessages/SipiMessages.scala:140]:
```suggestion
 * @param algorithm   the algorithm that was used to create the checksum of the file
 */
case class SipiUploadWithoutProcessingResponseEntry(
  filename: String,
  checksum: String,
  algorithm: String
```

**Review by mpro7 — APPROVED** [review]

File: webapi/src/it/scala/org/knora/webapi/it/v2/KnoraSipiIntegrationV2ITSpec.scala (hunk 1 above)
Comment (irinaschubert) [reply webapi/src/it/scala/org/knora/webapi/it/v2/KnoraSipiIntegrationV2ITSpec.scala]: No, it probably doesn't make much sense. I copied this test from above and just replaced the route. I removed this extra check now.

File: webapi/src/main/scala/org/knora/webapi/messages/store/sipimessages/SipiMessages.scala (hunk 2 above)
Comment (irinaschubert) [reply webapi/src/main/scala/org/knora/webapi/messages/store/sipimessages/SipiMessages.scala]: applied

**Pull request merged**